component/prometheus/operator/common/crdmanager.go

```diff
@@ -231,9 +231,12 @@ func (c *crdManager) runInformers(restConfig *rest.Config, ctx context.Context)
 		return fmt.Errorf("building label selector: %w", err)
 	}
 	for _, ns := range c.args.Namespaces {
+		// TODO: how should we handle this?
+		defaultNamespaces := map[string]cache.Config{}
+		defaultNamespaces[ns] = cache.Config{}
 		opts := cache.Options{
-			Scheme:     scheme,
-			Namespaces: []string{ns},
+			Scheme:            scheme,
+			DefaultNamespaces: defaultNamespaces,
```

Comment:
same question as with the other comment, how should we handle this change with the cache namespaces